bootloader/utils/util.h

```diff
@@ -41,4 +41,6 @@ u32 crc32c(const void *buf, u32 len);
 /* every 128 Bytes block. Intented only for Backup and Restore          */
 u32 memcmp32sparse(const u32 *buf1, const u32 *buf2, u32 len);
 
+char *itoa(int value, char *str, int base);
+
```

Comment:
We also use atoi() in main.c

Reply:
`atoi` is in the C standard, `itoa` is not

Reply:
i c